brewtils/__init__.py

```diff
@@ -26,71 +35,70 @@ def get_easy_client(**kwargs):
     return EasyClient(logger=logger, parser=parser, **get_bg_connection_parameters(**kwargs))
 
 
-def get_bool_from_kwargs_and_env(key, env_name, **kwargs):
-    """Gets a boolean value defaults to True"""
-    value = kwargs.get(key, None)
-    if value is None:
-        return os.environ.get(env_name, 'true').lower() != 'false'
-    elif isinstance(value, six.string_types):
-        return value.lower() != 'false'
-    else:
-        return bool(value)
+def get_bg_connection_parameters(cli_args=None, **kwargs):
+    """Convienence wrapper around ``load_config`` that returns only connection parameters
 
+    Args:
+        cli_args (list, optional): List of command line arguments for configuration loading
```

Comment:
`cli_args` is not used. I'm assuming it needs to get passed in to the `load_config` call?

Reply:
Yup. Oops.